packages/odyssey-icons/README.md

```diff
@@ -0,0 +1,3 @@
+# `@okta/odyssey-icons`
+
+The future home for Odyssey's icons.
```

Comment:
We should get into the habit of adding documentation whenever we're introducing new packages or adding functionality.

Here, it would be useful to:
- Describe the process of adding new icons
- Required commands needed for developers to run before opening a PR
- Update the available package list [here](https://github.com/okta/odyssey/#monorepo)

Reply:
We have a fast follow slated for next week to fully document this in both the package and the web docs. (Originally, we weren't going to ship anything other than the fresh package, but that didn't seem worthwhile once we got to the PR.)